<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_coder">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Coder</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_0">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CorerB_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_code">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Bin</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_0">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Isxodnyy</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_zaCode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>16</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Bin</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zacodirovanni</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_13">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DataB_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decoder">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decoder</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_data">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>16</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Bin</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_decode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Bin</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
